src/components/EventCard.js

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+
+const EventCard = ({ active, month, url, type, title, date }) => {
+    const cardContent = (
+        <Fragment>
+            <span
+                className={`tag ${type.toLowerCase()} icon`}
+            >{`${type[0].toUpperCase()}${type.substr(1)}`}</span>
```

Comment:
This would not catch cases such as `SOCIAL`, which would result in the same string. Would it be easier to just bring in a package for titleize that can handle this? It also makes the code a lot easier to read.

Reply:
Would the oneof proptype catch case?

Reply:
<img width="514" alt="screen shot 2018-07-24 at 7 06 13 pm" src="https://user-images.githubusercontent.com/1066856/43134451-b155ec78-8f74-11e8-99f1-5f0011cdf94e.png">

Seems like it